server/messages.go

```diff
@@ -527,12 +527,10 @@ func (s *Server) importAttachment(userID, messageID string, att *rfc822.Section)
 
 	var disposition, filename string
 
-	if header.Has("Content-Disposition") {
-		dispType, dispParams, err := mime.ParseMediaType(header.Get("Content-Disposition"))
-		if err != nil {
-			return proton.Attachment{}, fmt.Errorf("failed to parse attachment content disposition: %w", err)
-		}
-
+	if !header.Has("Content-Disposition") {
+		disposition = "attachment"
+		filename = "attachment.bin"
```

Comment:
Are you sure that adding this here won't cause any issues?

Reply:
When we ignored the failed attachments in the current process. The web/IMAP clients are still able to view the contents correctly.

Reply:
This is just in the test server. It has no impact on how bridge behaves against the real API.

If we fail to parse the attachments for whatever reason, we'll just fall back to the previous behaviour (full-message encryption).

Reply:
Won't that lead to different behavior between fake and real server?

Reply:
But that's the thing, our API servers don't consider this particular case an error (missing qoutes around file name) and handle this case properly.

Reply:
Changes pushed -- now if we fail to read either the content type or the content disposition, we fall back to the old full-message encrypter